Samples/1c-multiple-input-swaggers/Client/IStorageManagementClient.cs

```diff
@@ -0,0 +1,63 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.1.0
```

Comment:
Thanks, I wasn't sure if it was my change that did this but multiple regens gave this same diff